.swcrc

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "test": ".*\\.ts$",
+    "env": {
+      "targets": "node >= 16"
```

Comment:
v14 isn't supported?

Reply:
Thanks for spotting this. I imported the configuration from a personal project where I'm a little less gentle.
But we can definitely support Node.js 14 here. At least up to April when it goes EOL.

Reply:
Lyra is not supported by Node v14 since `utils.ts > getOwnProperty()` uses the **Object.hasOwn** method, which is not compatible with Node v14.

Reply:
Oh thanks, didn't know that. I'll add an `engines` property to `package.json` then.

Reply:
Hey @ShogunPanda, what about replacing the `Object.hasOwn` method so that Lyra can be compatible with Node versions <16?

Reply:
@micheleriva Can you remind me why we did include this? I remember it was a bug fix but I don't remember which. :)

Reply:
Curious, what is the philosophy on the lowest node version to support determined?
Node 18 became the most recent LTS  (long term support) version in Oct 2022

Reply:
Yes, you are right, 18 is the Active LTS.
But 16 is a Maintenance LTS supported up to next September.

Moreover, we don't strictly follow Node release cycle, we just try to support as many versions as possibile without stagnating or doing dirty hacks.

Reply:
Hey, @ShogunPanda since https://github.com/LyraSearch/lyra/pull/225 has merged the target could be `node >= 14`.

Reply:
Noted. Changing it.